database/seeds/DatabaseSeeder.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+use Illuminate\Database\Seeder;
+
+class DatabaseSeeder extends Seeder {
+	/**
+	 * Seed the application's database.
+	 *
+	 * @return void
+	 */
+	public function run() {
+
+		\Illuminate\Support\Facades\DB::statement( 'SET FOREIGN_KEY_CHECKS=0' );
```

Comment:
1. We shouldn't truncate tables before seeding. The idea is - seeders could be run at any time of application evaluation without changing rest of data. So no deletions (at least no `truncates`) in seeders.
Check this https://laravel.com/docs/5.8/eloquent#other-creation-methods (and maybe read/review whole chapter as you need it almost everywhere)
2. Use Eloquent models for seeding.
2.